### DISCUSSION TITLE
N+1 queries detection tooling for Sequel

### DISCUSSION THREAD

**Discussion by myprivatealaska** [jeremyevans/sequel#2219] [General]
Are there any tools/gems for detecting N+1 queries for Sequel? Most options I found work with Active Record or MongoID, for example, [Bullet](https://github.com/flyerhzm/bullet).

**Comment by AlexWayfer** [comment]
Feature request in Bullet: https://github.com/flyerhzm/bullet/issues/71#issuecomment-50974133

List of Sequel Plugins: https://sequel.jeremyevans.net/plugins.html

There is one for N+1 problem: https://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/ForbidLazyLoad.html
